pkg/storage/cacher.go

```diff
@@ -306,26 +286,11 @@ func (c *Cacher) Watch(ctx context.Context, key string, resourceVersion string,
 }
 
 // Implements storage.Interface.
-func (c *Cacher) WatchList(ctx context.Context, key string, resourceVersion string, filter Filter) (watch.Interface, error) {
```

Comment:
Sorry - I'm super busy today and I'm taking half a day off and I don't have time to really confirm it.

And we were already blocked for almost 10 hours, and we need to unblock merge-queue ASAP.